File: pde_matlab/drr1.m
function [dri, dro] = drr1(r,v0, L, a, ka, na, b, kt, vl, kL, B)

v = 4 / 3 * pi * r .^3;

dri = (a /(4*pi) * 1./(1 + (r/ka).^na) * L/(kL+L));
dro = (B + b * r.^2)./(4*pi * r.^2) .* (v - v0)./(v - v0 + kt*vl);

File: pde_python/drr1.py
# -*- coding: utf-8 -*-
"""
Created on Wed Jun 26 10:39:36 2019
@author: Léo Meyer

"""

import numpy as np

def drr1(r,a,kr,nr,kL,B,b,kt,v0,vl,L)    :
	
    v = (4/3.) * np.pi * r **3

    dri = (a /(4*np.pi) * 1./(1 + (r/kr)**nr) * L/(kL+L))
    dro = (B + b * r**2)/(4*np.pi * r**2) * (v - v0)/(v - v0 + kt*vl)
       
    return (dri,dro)
	
	
